scripts/travis_install_mcore.sh

```diff
@@ -0,0 +1,4 @@
+git clone https://github.com/trailofbits/manticore.git
+cd manticore
+pip install --no-binary keystone-engine -e .[dev]
```

Comment:
we also need to have a line like this here: https://github.com/trailofbits/manticore/blob/master/.travis.yml#L20
this is because the caching works by literally packing up the entire python directory where packages are installed. since manticore itself is installed as a package, it also gets cached, so on subsequent runs of these tests, without manually uninstalling manticore, this pip install line will hit the cache and use the old, cached manticore

the cache used to optimize dependency install time, however since we don't build unicorn or capstone from source anymore, this may not actually be necessary